sdk/nodejs/src/open-feature/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './DevCycleProvider'
+export * from './DevCycleMigrationStrategy'
```

Comment:
this ties the multi provider requirement to importing anything from the `open-feature` directory

Reply:
ah oops my bad meant to delete that line